src/components/list/index.stories.mdx

```diff
@@ -0,0 +1,25 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import List from './index.tsx';
+
+<Meta
+  title="Data Display/List"
```

Comment:
There is already a group named "Data display" further up in the "Patterns" navigation. Not sure what to do but it confuses me a bit

Reply:
https://automotivesmg.slack.com/archives/C03GK939L3V/p1665476814092119